web/src/components/snapshots/SnapshotDetails.jsx

```diff
@@ -311,8 +311,8 @@ class SnapshotDetails extends Component {
 
   renderShowAllVolumes = (volumes) => {
     return volumes.map((volume) => {
-      const diffMinutes = dayjs(volume?.finished).diff(
-        dayjs(volume?.started),
+      const diffMinutes = dayjs(volume?.finishedAt).diff(
+        dayjs(volume?.startedAt),
```

Comment:
started and finished never existed in the volumes object. 
converting more of these jsx to typescript files will help catch more of these